app/src/main/java/com/example/playlistmaker/search/ui/SearchFragment.kt

```diff
@@ -88,12 +82,14 @@ class SearchActivity : AppCompatActivity() {
         binding.etSearch.setOnEditorActionListener { _, actionId, _ ->
```

Comment:
Для скрытия клавиатуры у меня сработало вообще убрать этот код

```suggestion
        /* binding.etSearch.setOnEditorActionListener { _, actionId, _ ->
        if (actionId == EditorInfo.IME_ACTION_DONE) {
                val inputMethodManager =
                    getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
                inputMethodManager?.hideSoftInputFromWindow(binding.etSearch.windowToken, 0)

                    requireContext().getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
                inputMethodManager.hideSoftInputFromWindow(binding.etSearch.windowToken, 0)
                binding.etSearch.clearFocus()
                viewModel.startSearch()
                true
            } else false
            } else {
                false
            }
        }*/
```

при этом поиск при изменениях продолжает работать, так как ниже установлен TextWatcher

Reply:
И хорошо бы крестик для очистки поискового запроса

![Screenshot_20230918_215946](https://github.com/Andreydev1/PlaylistMaker_project/assets/36837067/9e4c12fb-830f-4e31-9b56-ad1058769a31)